### PR TITLE
Add chocolatey and scoop installation in docs

### PR DESCRIPTION
**Pull request by sitiom**
workspacer is available in Scoop, and recently I have started to maintain a [Chocolatey package of workspacer](https://community.chocolatey.org/packages/workspacer). It might be good to add these installation methods to the docs.

### CODE REVIEW

**Comment by josteink** [comment]
That's absolutely awesome and I definitely welcome this initiative￼!

That said, I think having the official release channel listed first and having third party distribution methods listed makes more sense. 

￼￼￼If you can order things that way, I will certainly approve of this PR. ￼

What do you say @rickbutton ? 🙂

**Comment by sitiom** [comment]
> That said, I think having the official release channel listed first and having third party distribution methods listed after that makes more sense.
> 
> ￼￼￼If you can order things that way, I will certainly approve of this PR. ￼

Sure, I will fix that.

**Review by josteink — APPROVED** [review]

**Comment by josteink** [comment]
PRs haven’t been merged in *months* and Rick haven’t been answering emails either. 

Right now it seems a fork is needed to keep workspacer going forwards. 

I’m discussing this in a issue here: https://github.com/rickbutton/workspacer/issues/211

Feel free to let me know your thoughts.

File: docs/content/quickstart.md [hunk 1]
```diff
@@ -6,7 +6,27 @@ type: docs
 
 # installing
 
-[download the latest installer or download the latest zip file and extract to a folder somewhere.](https://github.com/rickbutton/workspacer/releases/latest) workspacer automatically checks for updates.
+download the installer or the zip file from the [Releases](https://github.com/rickbutton/workspacer/releases/latest) page and install/unzip it anywhere you like.
```
Comment (dalyIsaac) [inline docs/content/quickstart.md:9]:
```suggestion
download the installer or the zip file from the [Releases](https://github.com/workspacer/workspacer/releases/latest) page and install/unzip it anywhere you like. 
```

**Review by dalyIsaac — CHANGES_REQUESTED** [review]
The URL needs to be updated, but otherwise LGTM!

**Comment by josteink** [comment]
@dalyIsaac: Your suggestion looks fixed now. Can you click “Review” and formally “approve” the PR? 🙂

**Review by dalyIsaac — APPROVED** [review]

**Comment by josteink** [comment]
Resolved the conflict and finally got this merged. Thanks for helping out everyone. 

We’re in a package-manager now 😎

**Pull request merged**